### PR TITLE
Add link to Multi Tenant Operator in Multi-Tenancy chapter

### PR DESCRIPTION
**Pull request by karl-johan-grahn**
### Description

Add link to Multi Tenant Operator in Multi-Tenancy chapter

### CODE REVIEW

**Review by dipesh-rawat — COMMENTED** [review]
@karl-johan-grahn Thanks for your contribution. It appears that this PR includes changes to files across multiple languages English (en) and Chinese (zh-cn). However, docs follows different processes for each localization, and we typically don't accept pull requests that impact multiple languages simultaneously. I would recommend modify only the primary website content (which means only English content) initially with this PR. After the merge of the English content, we could create separate pull request for the remaining language(s). I hope this suggestion is helpful.

**Comment by dipesh-rawat** [comment]
Explicit hold until this PR only covers one language.
_(Ok to unhold once that's done)_
/hold

File: content/en/docs/concepts/security/multi-tenancy.md [hunk 1]
```diff
@@ -462,6 +462,7 @@ listed below.
 
 #### Multi-team tenancy
 
+* [Multi Tenant Operator](https://docs.stakater.com/mto/)
 * [Capsule](https://github.com/clastix/capsule)
```
Comment (sftim) [inline content/en/docs/concepts/security/multi-tenancy.md:466]: Hi. These need to be in alphabetical order
```suggestion
{{% thirdparty-content %}}

* [Capsule](https://github.com/clastix/capsule)
* [Multi Tenant Operator](https://docs.stakater.com/mto/)
```
(the `thirdparty-content` shortcode is already in use on the page, but I recommend repeating it)

**Review by dipesh-rawat — COMMENTED** [review]
Thanks!

/lgtm

**Comment by sftim** [comment]
/label tide/merge-method-squash

OK to remove this label if @karl-johan-grahn squashes this down to 1 commit (which is tidier)

**Comment by karl-johan-grahn** [comment]
@dipesh-rawat @sftim squashed to 1 commit

**Review by dipesh-rawat — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**